src/main/algorithms/sort/heapSort.ts

```diff
@@ -91,6 +91,19 @@ export class HeapSort implements SortAlgorithmImplementation {
     }
 
     description(): string[] {
-        return ['Heapsort description']
+        return ['Heapsort ist eine Sortierverfahren, welches auf dem binären Heap als zentrale Datenstruktur aufbaut. \
+        Im ersten Schritt verändert Heapsort die Struktur der Liste in einer solchen Form, dass ein binärer Max-Heap \
```

Comment:
Sehr umständlich formuliert. Eventuell etwas wie: "Im ersten Schritt wird der binäre Heap in einen Max-Heap umgewandelt."